flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergStreamWriter.java

```diff
@@ -98,12 +102,21 @@ public String toString() {
         .toString();
   }
 
-  /** close all open files and emit files to downstream committer operator */
+  /**
```

Comment:
nit: this format change would probably fail the spotless check

Reply:
I think this should be a display problem, because I have not changed this.

Reply:
google-java-format would format this as a single line. please run spotless check. I would assume it will fail with this change.